### DISCUSSION TITLE
More than 1 email field causes the 2nd email field to display the value of the 1st email field

### DISCUSSION THREAD

**Discussion by ziming** [laravel/nova-issues#4255] [Q&A] [answered]
To reproduce

Email::make('Email'),
Email::make('Referrer Email'),

In index, detail and form view, referrer email will have same value as the 1st email field, even tho in database referred email is null.

**Comment by crynobone** [comment, marked as answer]
Use the following, similar to ID attribute for email is hardcoded to `email`, so for the second field you need to hardcode the attribute. 

```php
Email::make('Referrer Email', 'referrer_email'),
```
